pkg/avancement/service_manager_test.go

```diff
@@ -75,3 +77,32 @@ func mustAddCredentials(t *testing.T, repoURL string, a *git.Author) string {
 	}
 	return u
 }
+
+func TestRepositoryCloneErrorOmitsToken(t *testing.T) {
+	dstBranch := ""
```

Comment:
Yeah, so I had just copied the promote success test and was experimenting to see if this would fail, I've set it back to test-branch but indeed it makes no difference (perhaps this test could be trimmed)